src/wp-includes/abilities-api.php

```diff
@@ -0,0 +1,170 @@
+<?php
+/**
+ * Abilities API
+ *
+ * Defines functions for managing abilities in WordPress.
+ *
+ * @package WordPress
+ * @subpackage Abilities_API
+ * @since 6.9.0
+ */
+
+declare( strict_types = 1 );
+
+/**
+ * Registers a new ability using Abilities API.
+ *
+ * Note: Do not use before the {@see 'abilities_api_init'} hook.
```

Comment:
```suggestion
 * Note: Should only be used on the {@see 'abilities_api_init'} hook.
```

There will be errors if you register after the hook has fired, so I think this should be clearer